app/components/DocumentListItem.js

```diff
@@ -132,7 +131,7 @@ function DocumentListItem(props: Props, ref) {
             <>
               <Button
                 as={Link}
-                to={newDocumentUrl(document.collectionId, {
+                to={newDocumentUrl(document.computedCollectionId, {
```

Comment:
```suggestion
                to={newDocumentUrl(document.collectionId, {
```

As mentioned in chat, lets update `newDocumentUrl` so that undefined is valid first argument